### DISCUSSION TITLE
Libvips build from source using meson on Alpine 3.18

### DISCUSSION THREAD

**Discussion by michielswaanen** [libvips/libvips#3671] [General]
## Questions, enhancements, tips, etc.

Please use libvips discussions for other topics.

https://github.com/libvips/libvips/discussions

## Bug report

**Describe the bug**
I'm trying to build libvips from the source on Alpine using the meson build system. However, when all steps are completed the system can't find the vips command. Not sure what I'm doing wrong.

**To Reproduce**
1. Save Dockerfile
```Dockerfile:
FROM alpine:3.18

ARG VIPS_VERSION=8.14.5
ARG ALPINE_VERSION=3.18

RUN apk add xz meson \
    && wget https://github.com/libvips/libvips/releases/download/v${VIPS_VERSION}/vips-${VIPS_VERSION}.tar.xz \
    && tar -xf vips-${VIPS_VERSION}.tar.xz \
    && mv vips-${VIPS_VERSION} /tmp \
    && apk add \
    # For meson
    build-base \
    gobject-introspection-dev \
    # For libvips
    vips-dev \
    fftw-dev \
    openexr-dev \
    libgsf-dev \
    glib-dev \
    # liborc-dev                [no alpine equivalent found]
    # libopenslide-dev          [no alpine equivalent found]
    libaio-dev \
    libwebp-dev \
    libjpeg-turbo-dev \
    expat-dev expat \
    libexif-dev \
    tiff-dev \
    # libcfitsio-dev            [no alpine equivalent found]
    poppler-glib poppler-dev \
    librsvg-dev \
    pango-dev pango \
    # libopenjp2-7-dev          [no alpine equivalent found]
    libimagequant-dev

RUN cd /tmp/vips-${VIPS_VERSION} \
    && meson setup build-dir --prefix=/libvips \
    && cd build-dir \
    && meson compile \
    && meson install
```

2. Build Dockerfile:
```bash
$ docker build -f Dockerfile -t libvips-gh-issue-alpine .
```

3. Run Dockerfile
```bash
$ docker run -it libvips-gh-issue-alpine sh
~: vips
```

**Expected behavior**
vips command should work

**Actual behavior**
```bash
sh: vips: not found
```

**Environment**
(please complete the following information)
 - OS: Windows 10, WSL2, Docker Desktop
 - Vips: 8.14.5

**Comment by lovell** [comment]
> For libvips
> vips-dev \

If you're compiling libvips from source this is unnecessary (and can cause conflicts).

> --prefix=/libvips

It looks like you're specifying a custom `prefix`, so make sure the same path is also used at runtime - https://mesonbuild.com/Running-Meson.html#installing

**Reply by michielswaanen** [reply]
Thanks, removing the prefix did the trick. I see that you don't have an Alpine installation guide yet. I would be down to write it out if you're interested.
